stories/Molecules/Blocks/PublicationCard/publicationcard.scss

```diff
@@ -81,10 +79,25 @@
 }
 
 [dir='rtl'] {
-  .download-card__readmore {
-    .external-link {
-      margin-left: 0;
-      margin-right: $spacing-12;
+  .download-card {
+    &__readmore {
+      float: left;
+      .external-link {
+        margin-left: 0;
+        margin-right: $spacing-12;
+      }
+    }
+
+    &__description {
+      .grid-x>:not(:first-of-type) {
+        border-left: 0;
+        border-right: $spacing-02 solid #ced0d3;
```

Comment:
@neelesh-tippe Kindly do not use hardcoded color values.

Reply:
there is no variable for above color code.